src/backend/utils/mmgr/vmem_tracker.c

```diff
@@ -692,6 +700,8 @@ VmemTracker_UnregisterStartupMemory(void)
 		pg_atomic_sub_fetch_u32((pg_atomic_uint32 *) &MySessionState->sessionVmem,
 								startupChunks);
 
+	ResGroupProcAddStartupChunks(-startupChunks);
```

Comment:
Why do we need to call ResGroupProcAddStartupChunks here? What happens without this call?

Reply:
removed